.travis.yml

```diff
@@ -7,7 +7,7 @@ os:
 node_js:
   - 10
   - 8
-  - 6.2.0
+  - 6
```

Comment:
we'll want to keep that low 6.2.0 is the lowest I can run the full suite at without random seg faults (due to issues in earlier 6.0 releases). It's fine as Node official support is only whatever the latest 6 is but for nice coverage I like to test as far back as possible.

Reply:
yeah, that was just temporary. did you see my question regarding that toppic --> https://github.com/standard-things/esm/pull/550#issuecomment-410850973

I'll extend the skiplist to be conditional. not: all or nothing